src/platforms/java/migration.mdx

```diff
@@ -4,6 +4,28 @@ sidebar_order: 1000
 description: "Learn about migrating from io.sentry:sentry 1.x to io.sentry:sentry 3.x."
 ---
 
+## Migrating from `io.sentry:sentry` `5.x` to `io.sentry:sentry` `6.0.0`
```

Comment:
@maciejwalkowiak would you be able to help us add the migration items for the Java backend integrations? iirc:

> `SentryOptions` cannot be created anymore using `PropertiesProvider` with `SentryOptions#from` method. Use `ExternalOptions#from` instead and merge created object with `SentryOptions#merge`
> `SentryOptions` can merge properties from `ExternalOptions` instead of another instance of `SentryOptions`
> Feat: Relax TransactionNameProvider
> Ref: Remove deprecated and scheduled fields

Reply:
Change in `SentryOptions` does not affect directly backend integrations.

As far as I can see, only these notes should be added:

### Spring Boot

* `sentry.enable-tracing` property has been removed. To enable tracing, set `sentry.traces-sample-rate` property
* `SentryTracingFilter` can take a custom implementation of `TransactionNameProvider` when used with other web frameworks than Spring MVC

Reply:
Thanks, added those and a few more that I found out.